src/openrct2/localisation/StringIds.h

```diff
@@ -3911,8 +3911,10 @@ enum
 
     STR_COPY_INPUT_TO_CLIPBOARD = 6363,
 
+    STR_WINE_NOT_RECOMMENDED = 6368,
+
     // Have to include resource strings (from scenarios and objects) for the time being now that language is partially working
-    STR_COUNT = 32768
+    STR_COUNT = 32769
```

Comment:
Just adding a suggestion reflecting what Gymnasiast said.
```suggestion
    STR_COUNT = 32768
```